SECURITY.md

```diff
@@ -0,0 +1,212 @@
+includes encouraging responsible vulnerability research and disclosure. This
+policy sets out our definition of good faith in the context of finding and
+reporting vulnerabilities, as well as what you can expect from us in return.
+
+_If you have identified a vulnerability in Knative, please do not post an issue
+in the public issue tracker. Instead, please follow the responsible disclosure
+policy described in this document._
+
+## Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Extend [Safe Harbor](#safe-harbor) for your vulnerability research that is
+  related to this policy;
+- Work with you to understand and validate your report, including a timely
+  initial response to the submission;
+- Work to remediate discovered vulnerabilities in
+  [supported versions](#scope-and-supported-versions) in a timely manner;
+- Publicly disclose accepted vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to
+  report a unique vulnerability, and your report triggers a code or
+  configuration change.
+
+## Ground Rules
+
+To encourage vulnerability research and to avoid any confusion between
+good-faith hacking and malicious attack, we ask that you:
+
+- Play by the rules. This includes following this policy, as well as any other
+  relevant agreements, particularly the Knative
+  [Code of Conduct](https://knative.dev/community/contributing/code-of-conduct/).
+  If there is any inconsistency between this policy and any other relevant
+  terms, the terms of this policy will prevail;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting out-of-scope systems,
+  destroying data, and/or harming user experience;
+- Use only the [Official Channels](#reporting-a-vulnerability) to discuss
+  vulnerability information with us;
+- Keep the details of any discovered vulnerabilities confidential until they are
+  fixed, according to the Disclosure Policy;
+- Perform testing only on in-scope systems, and respect systems and activities
+  which are out-of-scope;
+- If a vulnerability provides unintended access to data: Limit the amount of
+  data you access to the minimum required for effectively demonstrating a Proof
+  of Concept; and cease testing and submit a report immediately if you encounter
+  any user data during testing, such as Personally Identifiable Information
+  (PII), Personal Healthcare Information (PHI), credit card data, or proprietary
+  information;
+- You should only interact with test accounts and Knative instances you own or
+  with explicit permission from the owner or account holder; and
+- Do not engage in extortion.
+
+## Safe Harbor
+
+When conducting vulnerability research according to this policy, we consider
+this research to be:
+
+- Authorized in accordance with the Computer Fraud and Abuse Act (CFAA) (and/or
```

Comment:
I based this doc on the USA version of terms from disclose.io, but perhaps we should use the generic version instead since knative and its users are international?

https://github.com/disclose/dioterms/blob/master/generic-core-terms.md#safe-harbor

Reply:
I added some language from the generic terms, and removed the restriction to "state" laws for this clause, but I left the USA-specific laws listed.